src/models/room.model.js

```diff
@@ -0,0 +1,23 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db');
+const { User } = require('./user.model');
+
+const Room = sequelize.define(
+  'room',
+  {
+    name: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false,
+    },
+  },
+  {
+    createdAt: false,
+    updatedAt: false,
+  },
+);
+
+Room.belongsTo(User);
+User.hasOne(Room);
```

Comment:
The `User.hasOne(Room)` association suggests that each user can only have one room. If users can belong to multiple rooms, consider using `User.hasMany(Room)` instead to accurately represent the relationship.